app/views/steps/case/has_codefendants/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
```

Comment:
No need for `path` here as we want to go back to the previous step so the path will be calculated automatically. Just `step_header` is enough like in other (intermediate) steps.